etc/sample_db.json

```diff
@@ -394,10 +394,6 @@
         "name": "line_length",
         "note": "Full width code, restricted line length.",
         "created": "2017-08-28"
-      }, {
-        "name": "sample_frame",
-        "note": "Frame live sample output.",
-        "created": "2017-09-05"
```

Comment:
Yes, this is how you can remove a flag from ``sample_db.json``. You even got the commas right! JSON is picky.

This will be applied the next time I create a sample database.